CHANGES.md

```diff
@@ -7,8 +7,10 @@
 * translate: Fix error handling when features cannot be read from reference sequence file. [#1168][] (@victorlin)
 * translate: Remove an unnecessary check which allowed for inaccurate error messages to be shown. [#1169][] (@victorlin)
 * frequencies: Previously, monthly pivot points calculated from the end of a month may have been shifted by 1-3 days. This is now fixed. [#1150][] (@victorlin)
+* Update package metadata for PyPI. [#1160][] (@corneliusroemer)
```

Comment:
I was surprised that this changelog entry doesn't say more specifically what changed. I think the utility of the changelog entry is pretty limited without that!

Reply:
Reworded in [force-push](https://github.com/nextstrain/augur/compare/8e61017a0b6d02c55ce80478285af1eb5f1ced44..bf991481f14c046e92453384fc7df874510b2639).

Reply:
:pray: